integration_tests/utils.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+CONTAINER='kw-integration'
+IMAGE='kw-raw'
+
+function run_unit_tests()
+{
+  # Run unit tests
+  podman exec "$CONTAINER" bash -c "./run_tests.sh"
+}
+
+function run_integration_tests()
+{
+  # Run integration tests
+  podman exec "$CONTAINER" bash -c "./run_integration_tests.sh"
+}
```

Comment:
tbh, I did not get the raison d'être of these functions... 

However, they gave me some ideas :)

We could organize the code as something like this:

```sh
run_integration_test.sh
integration_test/
 -> utils/
      -> Containerfile
      -> utils.sh
 kw_build_test.sh
 kw_version_test.sh
 ...
```

In the `run_integration_test.sh` you basically start the container before running any test and stop/clean after the test finishes, that way, each test does not need to deal with starting and stopping a VM; i.e., the test can assume the container is running and ready.

The `kw_version_test.sh` can take advantage of shunit and it can look like this:

```sh
include '../tests/utils.sh'

oneTimeSetUp()
{
  # Get the current branch name
  # Get the first line of the VERSION file
}

function test_check_if_kw_version_gives_the_correct_version()
{
   output=$(run_kw_command 'kw version')

   # compare $output with the expected value
}
```